descartes_tests/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(descartes_tests)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  descartes_moveit
+  descartes_planner
+  descartes_trajectory
+)
+
+catkin_package(
+  INCLUDE_DIRS
+    include
+  LIBRARIES
+    ${PROJECT_NAME}
+  CATKIN_DEPENDS
+    descartes_moveit
+    descartes_planner
+    descartes_trajectory
+)
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_library(${PROJECT_NAME}
+  src/cartesian_robot.cpp
+)
+
+add_dependencies(${PROJECT_NAME} ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+target_link_libraries(${PROJECT_NAME}
+  ${catkin_LIBRARIES}
+)
+
+install(TARGETS ${PROJECT_NAME}
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.h"
+)
+
+if(CATKIN_ENABLE_TESTING)
+  # Trajectory Unit Tests
+  catkin_add_gtest(${PROJECT_NAME}_trajectory_utest
+    test/trajectory/utest.cpp
+    test/trajectory/axial_symmetric_pt.cpp
+    test/trajectory/cart_trajectory_pt.cpp
+    test/trajectory/joint_trajectory_pt.cpp
+    test/trajectory/cartesian_robot_test.cpp
+  )
+  target_compile_definitions(${PROJECT_NAME}_trajectory_utest PUBLIC GTEST_USE_OWN_TR1_TUPLE=0)
```

Comment:
See docs [here](https://github.com/google/googletest/tree/master/googletest#choosing-a-tr1-tuple-library). Basically, I'm telling GTEST to use the c++11 `std::tuple`. A while ago (perhaps on Indigo?) I had trouble with a strange build error involving multiple versions of said class and this was the solution. This line was copied from the other tests.